app/client/reducers/auth/types.ts

```diff
@@ -45,128 +40,60 @@ export interface LoginSuccess {
-}
-export const REGISTER_FAILURE = "REGISTER_FAILURE";
-export interface RegisterFailure {
-  type: typeof REGISTER_FAILURE;
-  error: string;
-}
-export const REGISTER_INACTIVE = "REGISTER_INACTIVE";
-export interface RegisterInactive {
-  type: typeof REGISTER_INACTIVE;
+
+export enum ActivateActionTypes {
+  ACTIVATE_REQUEST = "ACTIVATE_REQUEST",
+  ACTIVATE_SUCCESS = "ACTIVATE_SUCCESS",
+  ACTIVATE_FAILURE = "ACTIVATE_FAILURE"
 }
-export type RegisterAction =
-  | RegisterRequest
-  | RegisterSuccess
-  | RegisterFailure
-  | RegisterInactive;
 
-export const ACTIVATE_REQUEST = "ACTIVATE_REQUEST";
 export interface ActivatePayload {
   body: {
     activation_token: string;
   };
 }
-export interface ActivateRequest {
-  type: typeof ACTIVATE_REQUEST;
-  payload: ActivatePayload;
-}
-export const ACTIVATE_SUCCESS = "ACTIVATE_SUCCESS";
-interface ActivateSuccess {
-  type: typeof ACTIVATE_SUCCESS;
-}
-export const ACTIVATE_FAILURE = "ACTIVATE_FAILURE";
-interface ActivateFailure {
-  type: typeof ACTIVATE_FAILURE;
-  error: string;
+
+export enum ResetPasswordEmailActionTypes {
+  REQUEST_RESET_REQUEST = "REQUEST_RESET_REQUEST",
+  REQUEST_RESET_SUCCESS = "REQUEST_RESET_SUCCESS",
+  REQUEST_RESET_FAILURE = "REQUEST_RESET_FAILURE"
 }
-export type ActivateAction =
-  | ActivateRequest
-  | ActivateSuccess
-  | ActivateFailure;
 
-export const REQUEST_RESET_REQUEST = "REQUEST_RESET_REQUEST";
 export interface ResetEmailPayload {
   body: {
     email: string;
   };
 }
-export interface ResetEmailRequest {
-  type: typeof REQUEST_RESET_REQUEST;
-  payload: ResetEmailPayload;
-}
-export const REQUEST_RESET_SUCCESS = "REQUEST_RESET_SUCCESS";
-interface ResetEmailSuccess {
-  type: typeof REQUEST_RESET_SUCCESS;
-}
-export const REQUEST_RESET_FAILURE = "REQUEST_RESET_FAILURE";
-interface ResetEmailFailure {
-  type: typeof REQUEST_RESET_FAILURE;
-  error: string;
+
+export enum ResetPasswordActionTypes {
+  RESET_PASSWORD_REQUEST = "RESET_PASSWORD_REQUEST",
+  RESET_PASSWORD_SUCCESS = "RESET_PASSWORD_SUCCESS",
+  RESET_PASSWORD_FAILURE = "RESET_PASSWORD_FAILURE"
 }
-export type ResetEmailAction =
-  | ResetEmailRequest
-  | ResetEmailSuccess
-  | ResetEmailFailure;
 
-export const RESET_PASSWORD_REQUEST = "RESET_PASSWORD_REQUEST";
 export interface ResetPasswordPayload {
   body: {
     new_password: string;
     reset_password_token: string;
     old_password: string;
   };
 }
-interface ResetPasswordRequest {
-  type: typeof RESET_PASSWORD_REQUEST;
-  payload: ResetPasswordPayload;
-}
-export const RESET_PASSWORD_SUCCESS = "RESET_PASSWORD_SUCCESS";
-interface ResetPasswordSuccess {
-  type: typeof RESET_PASSWORD_SUCCESS;
-}
-export const RESET_PASSWORD_FAILURE = "RESET_PASSWORD_FAILURE";
-interface ResetPasswordFailure {
-  type: typeof RESET_PASSWORD_FAILURE;
-  error: string;
-}
-export type ResetPasswordAction =
-  | ResetPasswordRequest
-  | ResetPasswordSuccess
-  | ResetPasswordFailure;
 
 export interface AdminUser {
```

Comment:
NIT: Given all other interfaces for payloads include "Payload" in variable name, this should follow the same pattern... i.e. `AdminUserPayload`

Reply:
Pretty sure I wrote this part of the code lol

Reply:
actually just realized this doesn't make sense. scratch that